### PR TITLE
Added text to hygienekitsCard and price

### PR DESCRIPTION
**Pull request by Sukhamjot-Singh**
This PR tries to resolve issue #13.
Please Review.

### CODE REVIEW

File: pages/HygieneKitsCard.js [hunk 1]
```diff
@@ -3,10 +3,22 @@ function HygieneKitsCard() {
...
+            Each hygiene kit allows an unhoused individual to freshen up and
+            feel more confident about their hygiene for work, school, or even
+            just to have a better day.
+          </p>
+          <p>
+            <a
+              class="btn btn-primary"
+              href="https://www.amazon.com/96-Kits-Hygiene-Travel-Charity/dp/B0881YKMBL"
+            >
+              96 Hygiene Kits for 153$
```
Comment (audreyfeldroy) [inline pages/HygieneKitsCard.js:19]:
```suggestion
              96 Hygiene Kits for $153
```

**Comment by audreyfeldroy** [comment]
Hey @Sukhamjot-Singh, I got caught up in meetings but am back now. Thank you so much for submitting this second PR: it's incredibly kind of you and it means a lot to us.

@pydanny can you approve the deploy preview? 😄

**Comment by pydanny** [comment]
Done!

**Review by pydanny — APPROVED** [review]
Thank you so much for the contribution!

**Comment by pydanny** [comment]
The code is solid, atomic, and just what was needed.

**Comment by Sukhamjot-Singh** [comment]
Thank you both of you.
I am actually new to openSource.
This project had doable issues.
Hope to contribute more.

**Pull request merged**